assets/src/edit-story/components/canvas/pagenav/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app';
+
+const Wrapper = styled.div`
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+	height: 100%;
+	color:  ${ ( { theme } ) => theme.colors.fg.v1 };
+`;
+
+const Space = styled.div`
+	flex: 0 1 ${ ( { isNext } ) => isNext ? 60 : 115 }px;
+`;
+
+const NavButton = styled.button`
+	display: block;
+	background: transparent;
+	border: none;
+	width: 40px;
+	height: 40px;
+	&:focus {
+		outline: 1px dotted -webkit-focus-ring-color
```

Comment:
The default behavior seems to be 5px solid outline. Perhaps it can be just left to default then for now and be handled later with consistency considering other buttons, too.